### ISSUE TITLE
YOLOv8 GPU RTX4090 train all loss and P/R/map are 0

### ISSUE THREAD

**Issue by Kukki0803** [ultralytics/ultralytics#9947]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

![微信截图_20240410223059](https://github.com/ultralytics/ultralytics/assets/72021510/7c07e5db-7049-491b-a588-7671fbbd425f)
![微信截图_20240410223125](https://github.com/ultralytics/ultralytics/assets/72021510/2c43569b-b907-4f9d-98fc-6d18230e2d52)

**Comment by glenn-jocher** [comment]
@Kukki0803 hey there! It seems like you're experiencing an issue where all losses and performance metrics are zero during training on an RTX4090 GPU. This is a bit unusual. A few things you might want to check:

1. **Dataset:** Ensure your dataset is correctly labeled and paths in your data YAML are correct. Mistakes here could lead to the model not seeing any data.
2. **CUDA, PyTorch, and GPU Drivers**: Verify that your environment is correctly set up, including up-to-date GPU drivers, CUDA version compatible with your PyTorch installation, and PyTorch itself. Sometimes mismatches here can cause unexpected behaviors.
3. **Batch Size & Learning Rate**: If too low, the model might not be effectively learning. Try adjusting these parameters.
4. **Configuration File**: Double-check your model config file for any anomalies that might cause this.

If the problem persists, consider running a very small subset of your data to see if the issue is consistent across different sets. Here's a quick example of how to initiate training with a basic check for dataset path:

```bash
# Assuming coco128.yaml is your dataset config file
yolo train data=coco128.yaml model=yolov8n.pt epochs=3 imgsz=640
```

This trains a YOLOv8 model for 3 epochs on a dataset, ideally indicating if the issue persists across different conditions. If you could share any error messages or more specific details about your setup, that would also be helpful for deeper investigation. 🕵️‍♂️
